File: 05_hash_tables/java/01_price_of_groceries.java
import java.util.*;
import java.lang.*;
import java.io.*;


class grokkingAlgorithms
{	

	public static void main (String[] args) throws java.lang.Exception
	{
        HashMap<String, Double> book = new HashMap<String, Double>();
        book.put("apple", 0.67);
        book.put("milk", 1.49);
        book.put("avocado", 1.49);

        Set set = book.entrySet();
        Iterator iterator = set.iterator();
        while(iterator.hasNext()) {
            Map.Entry mentry = (Map.Entry)iterator.next();
            System.out.print(mentry.getKey() + " ");
            System.out.println(mentry.getValue());
         }
	}
}

File: 05_hash_tables/javascript/01_price_of_groceries.js
var book = new Map();
// an apple costs 67 cents
book["apple"] = 0.67;
// milk costs $1.49
book["milk"] = 1.49;
book["avocado"] = 1.49;

for ( [key, value]  of Object.entries(book)) {
  console.log(key, value);

}
